<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>641</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labViewSize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>565</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Graphics View坐标，左上角总是（0,0），宽度=，长度=</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabSceneRect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>565</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>QGraphicsView::sceneRect=()</string>
     </property>
    </widget>
   </widget>
   <widget class="GraphiceView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>721</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphiceView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphiceview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
